### ISSUE TITLE
Merge into retrofit/retrofit-converters

### ISSUE THREAD

**Issue by GMCristiano** [JakeWharton/retrofit2-kotlinx-serialization-converter#48]
Any plans on adding your converter into the retrofit-converters compilation? 

https://github.com/square/retrofit/tree/master/retrofit-converters

**Comment by JakeWharton** [comment]
Not until it only relies on stable APIs. Currently the API to look up a serializer from a Type is marked as experimental. Once that or an alternative is stable it will move!
